partner_financial_risk/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================
+Partner Financial Risk
+======================
+
+Adds a new page in partner to manage the partner's *Financial Risk*
+If any limit is exceed the the partner is blocked to confirm sale orders.
```

Comment:
Can you elaborate the rationale behind your calculation of the risk?
Is it based on some documentation/best practices?
Detailing the criteria and / or documenting with URL/guide is necessary for this kind of module
Checking the code is not that clear and gives me the feeling that this is specific for one customer/accountant PoV.
I am not saying it is wrong but we need modules that are generic or related to best practices and we need to document them if necessary.